Sources/Functions/Block.swift

```diff
@@ -23,14 +23,11 @@
 /// Closure with custom arguments and return value.
 public typealias Block<Input, Output> = (Input) -> Output
```

Comment:
maybe rename to `Closure<Input, Output>`? name VoidBlock came from legacy objc projects. we can keep `VoidBlock` as alias to () -> Void, but rename all others